src/runtime/composables/local/useAuth.ts

```diff
@@ -163,8 +163,16 @@ async function getSession(getSessionOptions?: GetSessionOptions): Promise<Sessio
 
 async function signUp(credentials: Credentials, signInOptions?: SecondarySignInOptions, signUpOptions?: SignUpOptions) {
   const nuxt = useNuxtApp()
+  const runtimeConfig = useRuntimeConfig()
+  const config = useTypedBackendConfig(runtimeConfig, 'local')
+
+  const signUpEndpoint = config.endpoints.signUp
+
+  if (!signUpEndpoint) {
```

Comment:
Hey @root5427 👋 

Thanks for the contribution. I was wondering if we may want to add a `console.warn` message here that the endpoint is disabled? I feel like if someone disabled the endpoint by accident and then attempts to call the function regardless it would be nice to give a short warning to make it eaiser to find out why the function is not working 😄

Reply:
Hi, Zoey.
Great suggestion, thank you very much. I will commit corresponding changes 👍